stactools_pipelines/pipelines/aws_noaa_oisst_avhrr_only/test_collection.py

```diff
@@ -12,7 +12,7 @@
 
 def test_create_collection():
     collection = module_create_collection()
-    assert type(collection) == pystac.collection.Collection
+    assert type(collection) is pystac.collection.Collection
```

Comment:
I made this change in my PR (#45) but this is a perfect case for `isinstance(collection, pystac.Collection)